src/components/InfoFooter.vue

```diff
@@ -56,11 +56,13 @@
 </template>
 
 <script>
+import { bgColorClass } from '@/lib/BgColorClass'
+
 export default {
   name: 'InfoFooter',
   data() {
     return {
-      environment: import.meta.env.MODE,
+      environment: import.meta.env['VITE_ENVIRONMENT'],
```

Comment:
Is there a specific reason we're moving away from [`MODE`](https://vitejs.dev/guide/env-and-mode#env-variables)?

It looks like having a `VITE_ENVIRONMENT=production` in `.env.production` and `VITE_ENVIRONMENT=uat` in `.env.uat` is a bit redundant. Vite builds and [attaches the correct environment](https://vitejs.dev/guide/env-and-mode#modes) name when running: 

```bash
vite build --mode <environment name>
```

This is just me getting some clarifications :)

Reply:
Does 'import.meta.env.MODE' only return 'production' & 'development'? can't distinguish 'uat' & 'production'?

Reply:
Yes. Node only has development and production.

Reply:
I think it depends on the build script. So, if you build your application with `vite build --mode uat`, it would attach the configurations (at build time) in `.env.uat` (take a look at the [documentation](https://vitejs.dev/guide/env-and-mode#modes) how they've utilised `staging`).

I'm not well versed in this area. Let's bring some experts in 😄 @seenanair

Reply:
Yes, for uat I think we need to specifically set the mode. But, I do not know we are using that for uat deployment.
 So either we can  change script in deployment project for uat build which allows mode to be used , which allows meta.env.MODE to be used across all environments or  to keep this VITE_ENVIRONMENT flag and manage it

Reply:
(Sorry @stevieing, didn't see your reply when I was typing [this reply](https://github.com/sanger/traction-ui/pull/1762#discussion_r1660737927)).

So, are we building the UI using something other than Vite? In [`package.json`](https://github.com/sanger/traction-ui/blob/3208cb24e294df6f8b1f0ae126b634c2dbc907a9/package.json#L12) it looks like we're using `vite build` though.

Reply:
We would have to make changes to the github action as we build on release not in the deployment project. Not entirely sure how we would do this at this stage as we would need to set the mode when we build a release so we would need to do it conditionally either for develop or master.